src/pages/characters/[[...params]].js

```diff
@@ -93,6 +93,7 @@ export default function Characters() {
         columns={columns}
         data={characters}
         onRowClick={({ id }) => handleOpenForm(id)}
+        editableRow
```

Comment:
This is not a good prop name for a boolean. Booleans should be prefixed with words like "is", "has", "should" etc,  so you can tel by the name it's a boolean. This makes reading code easier